libs/blocks/adobetv/adobetv.js

```diff
@@ -12,12 +12,16 @@ export default function init(a) {
       anchorTag: a,
     });
   } else {
+    const textContent = a?.textContent?.trim();
+    const titleFromText = textContent?.includes('|') ? textContent.split('|')[1].trim() : null;
+    const title = titleFromText || 'Adobe Video Publishing Cloud Player';
```

Comment:
NIT: you can remove the above ternary
```suggestion
    const title = textContent.split('|')?.[1]?.trim() || 'Adobe Video Publishing Cloud Player';
```